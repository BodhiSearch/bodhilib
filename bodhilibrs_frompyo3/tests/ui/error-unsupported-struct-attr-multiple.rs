use frompyo3::FromPyO3;

#[derive(FromPyO3)]
#[frompyo3(name = "Resource")]
#[frompyo3(foo = "bar")]
#[frompyo3(baz = "caz")]
struct TestUnsupportedStructAttrMultiple {
  path: String,
}

fn main() {}
