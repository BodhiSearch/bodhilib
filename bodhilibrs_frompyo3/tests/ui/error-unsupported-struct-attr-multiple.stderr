error: Multiple attributes not supported
 --> tests/ui/error-unsupported-struct-attr-multiple.rs:5:1
  |
5 | #[frompyo3(foo = "bar")]
  | ^^^^^^^^^^^^^^^^^^^^^^^^

error: Multiple attributes not supported
 --> tests/ui/error-unsupported-struct-attr-multiple.rs:6:1
  |
6 | #[frompyo3(baz = "caz")]
  | ^^^^^^^^^^^^^^^^^^^^^^^^
